src/NuGet.Core/NuGet.Frameworks/DefaultFrameworkMappings.cs

```diff
@@ -325,7 +326,15 @@ public IEnumerable<OneWayCompatibilityMappingEntry> CompatibilityMappings
                             new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.Windows, FrameworkConstants.MaxVersion)),
                             new FrameworkRange(
                                 new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.WinRT, FrameworkConstants.EmptyVersion),
-                                new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.WinRT, new Version(4, 5, 0, 0))))
+                                new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.WinRT, new Version(4, 5, 0, 0)))),
+
+                        // NetCoreApp projects support NetStandard
+                        new OneWayCompatibilityMappingEntry(new FrameworkRange(
+                            new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.NetCoreApp, new Version(1, 0, 0, 0)),
+                            new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.NetCoreApp, FrameworkConstants.MaxVersion)),
+                            new FrameworkRange(
+                                FrameworkConstants.CommonFrameworks.NetStandard10,
```

Comment:
I'm trying to understand what this is saying. Is it saying it supports the entire range of netstandard10 to netstandard15? I was kind of only expecting to see a mapping to netstandard15 and the compat with the earlier netstandard versions would just happen. However I admit I very little about how the magic in this file actually works.